server/src/main/resources/openapi/schemas/oai-pmh-client-status.json

```diff
@@ -17,18 +17,52 @@
             "type" : "string",
             "description" : "Error message for last error"
           },
+          "lastActiveTimestamp" : {
+            "description" : "timestamp when the job was last active (last successful ingest)",
+            "type" : "string",
+            "format" : "date-time"
+          },
+          "lastRecsPerSec" : {
+            "description" : "Average records per second for the last run",
+            "type" : "integer"
+          },
+          "lastRunningTime" : {
```

Comment:
Since the status for operator's convenience it would be better to report this as a human-readable duration string (XX days YY hrs ZZ mins TT secs) with something like `String.format("%02d days %02d hrs %02d mins %02d secs", s / (24 * 3600), (s % (24* 3600)) / 24, (s % 3600) / 60, (s % 60));`. Could we change this?

Reply:
ok